File: src/main/java/ru/ioleksiv/telegram/bot/core/model/telegram/objects/type/files/PhotoSize.java
package ru.ioleksiv.telegram.bot.core.model.telegram.objects.type.files;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @link https://core.telegram.org/bots/api#photosize
 */
public class PhotoSize {
    /**
     * file_id	String	Unique identifier for this files
     */
    @JsonProperty("file_id")
    private String fileId;
    /**
     * width	Integer	Photo width
     */
    @JsonProperty("width")
    private Integer width;
    /**
     * height	Integer	Photo height
     */
    @JsonProperty("height")
    private Integer height;
    /**
     * file_size	Integer	Optional. File size
     */
    @JsonProperty("file_size")
    private Integer fileSize;

    public String getFileId() {
        return fileId;
    }

    public void setFileId(String fileId) {
        this.fileId = fileId;
    }

    public Integer getWidth() {
        return width;
    }

    public void setWidth(Integer width) {
        this.width = width;
    }

    public Integer getHeight() {
        return height;
    }

    public void setHeight(Integer height) {
        this.height = height;
    }

    public Integer getFileSize() {
        return fileSize;
    }

    public void setFileSize(Integer fileSize) {
        this.fileSize = fileSize;
    }
}

File: server-api/src/main/kotlin/io/github/oleksivio/tl/kbot/server/api/objects/std/files/PhotoSize.kt
package io.github.oleksivio.tl.kbot.server.api.objects.std.files

import com.fasterxml.jackson.annotation.JsonProperty
import io.github.oleksivio.tl.kbot.server.api.model.ITelegram

/**
 *  [PhotoSize](https://core.telegram.org/bots/api/#photosize)
 */
data class PhotoSize(
        /**
         * file_id String Unique identifier for this files
         */
        @JsonProperty("file_id")
        var fileId: String? = null,
        /**
         * To setup filter:
         *
         *  IntegerFilter width width Integer Photo width
         */
        @JsonProperty("width")
        var width: Int? = null,
        /**
         * To setup filter:
         *
         *  IntegerFilter height height Integer Photo height
         */
        @JsonProperty("height")
        var height: Int? = null,
        /**
         * To setup filter:
         *
         *  IntegerFilter fileSize file_size Integer Optional. File size
         */
        @JsonProperty("file_size")
        var fileSize: Int? = null
) : ITelegram
